docs/site/migration/models/mixins.md

```diff
@@ -6,8 +6,805 @@ sidebar: lb4_sidebar
 permalink: /doc/en/lb4/migration-models-mixins.html
 ---
 
-{% include note.html content="
-This is a placeholder page, the task of adding content is tracked by the
-following GitHub issue:
-[loopback-next#3951](https://github.com/strongloop/loopback-next/issues/3951)
-" %}
+## Introduction
+
+This document will guide you in migrating custom model mixins, and custom method
+mixins in LoopBack 3 to their equivalent implementations in LoopBack 4.
+
+For an understanding of how models in LoopBack 3 are now architectually
+decoupled into 3 classes (model, repository, and controller) please read
+[Migrating custom model methods](./methods.md).
+
+In LoopBack 3, it was easy to add property mixins and method
+[mixins](https://loopback.io/doc/en/lb3/Defining-mixins.html).
+
+In LoopBack 4, it is also easy and is accomplished by using a mixin class
+factory function.
+
+## LoopBack 3 Approach To Creating A Model Property Mixin
+
+In LoopBack 3, a developer is able to create a model property mixin by:
+
+- placing the mixin logic in a file in a mixins directory
+- updating the server/model-config.json file with the mixin directory location
+- updating the model's json file to include the mixin's name and a boolean
```

Comment:
I love ❤️ these short list of steps that you are showing in different places of the document, they make it easier to grasp the overal structure of this guide 👍